src/main/java/org/example/lesson10task6/BirthDate.java

```diff
@@ -0,0 +1,22 @@
+package org.example.lesson10task6;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoField;
+
+public class BirthDate {
+    public static int age(LocalDate dateOfBirth) {
+        LocalDate dateNow = LocalDate.now();
+        int age = dateNow.getYear() - dateOfBirth.getYear();
+        return age;
```

Comment:
Przy wyliczaniu wieku weź proszę pod uwagę też dzień urodzenia, czyli na 27.01.2023 jeżeli ktoś urodził się 18.01.2000 to ma 23 lata ale jeżeli urodził się 18.09.2000 to ma jeszcze 22 lata :)